hugo.yaml

```diff
@@ -34,13 +34,13 @@ menu:
         teaser: >
           Discover our recommendations for developing benchmarks.
 
-    #- name: Projects
-      #pageRef: /projects
-      #weight: 3
-      #params:
-        #type: playbook
-        #teaser: >
-          #Discover our recommendations for project management.
+    - name: Projects
+      pageRef: /projects
```

Comment:
Directory is named `project`: Either `git mv content/project content/projects` or change the link here to `/project` instead of `/projects`